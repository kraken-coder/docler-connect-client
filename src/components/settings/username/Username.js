import React from 'react'

import UserNameInput from '../../global/UserNameInput'

import '../style.scss'

const Username = () => (
  <div className="settings-username">
    <p className="username-label">Change Username</p>
    <UserNameInput />
  </div>
)

export default Username
