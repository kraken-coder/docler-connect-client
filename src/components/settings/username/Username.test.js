import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import Username from './Username'

test('Renders Without Crashing', () => {
  const Wrapper = shallow(<Username />)
  expect(toJSON(Wrapper)).toMatchSnapshot()
})
