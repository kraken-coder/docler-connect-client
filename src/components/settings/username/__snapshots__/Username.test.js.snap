// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Without Crashing 1`] = `
<div
  className="settings-username"
>
  <p
    className="username-label"
  >
    Change Username
  </p>
  <UserNameInput
    onChange={null}
    value={null}
  />
</div>
`;
